### PR TITLE
Lacy gondola: Private icon

### PR DESCRIPTION
**Pull request by whimsicallyson**
## Links
* https://lacy-gondola.glitch.me/

## Changes:
* Some of this was already changed in previous PRs (we had private icons as badges, emoji, and components IIRC) but now they use Icon when its just the image, and wrapped in our component when interactive (tooltip, toggle)

## How To Test:
* Check out your user page, with private projects and collections holding private projects
* Check out the ~ page of a private project you belong to
* Check out the ~ page of a private project you *don't* belong to 

## Feedback I'm looking for:
Everything look good, work, toggle appropriately?

### CODE REVIEW

**Comment by sarahzinger** [comment]
looks pretty good, i see there's a bug with the padding of the lock icon for the badge you might want to look at!
Screen Shot 2019-09-13 at 1 37 38 PM

**Comment by whimsicallyson** [comment]
@sarahzinger @ttseng updated badge padding! (it needs to be slightly different depending on whether its clickable, because of the border)

File: src/components/private-badge/index.js [hunk 1]
```diff
@@ -9,29 +10,22 @@ import styles from './styles.styl';
...
-  <TooltipContainer
-    type="action"
-    tooltip={isPrivate ? privateText : publicText}
-    target={
-      <HiddenCheckbox value={isPrivate} onChange={setPrivate}>
-        <PrivateIcon isPrivate={isPrivate} className={styles.button} label={privateText} />
-      </HiddenCheckbox>
-    }
-  />
+  <span className={classnames(styles.button, styles.projectBadge, isPrivate ? styles.private : styles.public)}>
```
Comment (sarahzinger) [inline src/components/private-badge/index.js]: i'm curious what the thought is in putting the span here vs inside the target. not sure it matters but if it does thought I'd ask to better understand :)

Comment (whimsicallyson) [reply src/components/private-badge/index.js]: the styling didn't come through properly that way? 🤷‍♀

**Pull request merged**